From 542a0cdd7088cba5c3ba1bc7927d40f3419113e6 Mon Sep 17 00:00:00 2001
From: Arulselvan M <arulselvan.m@intel.com>
Date: Thu, 4 Dec 2014 12:46:00 +0530
Subject: [PATCH 11/13] [REVERT_ME]: ASoC: Intel: Force disable audio clocks
 during s3 transition

Issue: GMINL-4531

This disables the platform audio clock during resume path.
Added as workaround until the fix.
This patch needs to reverted once we have proper fix.

Change-Id: I35e60d8021b1f194f1ad5d109af9cac2d111a0c2
Signed-off-by: Arulselvan M <arulselvan.m@intel.com>
---
 sound/soc/intel/board/cht_bl_dpcm_rt5672.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/sound/soc/intel/board/cht_bl_dpcm_rt5672.c b/sound/soc/intel/board/cht_bl_dpcm_rt5672.c
index 6b9571c..611a9f1 100644
--- a/sound/soc/intel/board/cht_bl_dpcm_rt5672.c
+++ b/sound/soc/intel/board/cht_bl_dpcm_rt5672.c
@@ -851,6 +851,7 @@ static int snd_cht_prepare(struct device *dev)
 static void snd_cht_complete(struct device *dev)
 {
 	pr_debug("In %s\n", __func__);
+	vlv2_plat_configure_clock(VLV2_PLAT_CLK_AUDIO, PLAT_CLK_FORCE_OFF);
 	snd_soc_resume(dev);
 }
 
-- 
1.7.9.5

